app/workers/classifications_dump_worker.rb

```diff
@@ -8,10 +8,12 @@ class ClassificationsDumpWorker
 
   sidekiq_options queue: :data_high
 
-  def perform_dump(obfuscate_private_details=true)
+  def perform_dump(private=true)
```

Comment:
`private=true` sounds more like "this is a private dump" or "yes, include private stuff". Maybe `anonymize=true` if you're searching for something shorter?